packages/desktop-electron/findOpenSocket.js

```diff
@@ -1,33 +1,7 @@
-const ipc = require('node-ipc');
-
-ipc.config.silent = true;
-
-function isSocketTaken(name, fn) {
-  return new Promise((resolve, reject) => {
-    ipc.connectTo(name, () => {
-      ipc.of[name].on('error', () => {
-        ipc.disconnect(name);
-        resolve(false);
-      });
-
-      ipc.of[name].on('connect', () => {
-        console.log('connected');
-        ipc.disconnect(name);
-        resolve(true);
-      });
-    });
-  });
-}
+const { getRandomPort } = require('get-port-please');
 
 async function findOpenSocket() {
```

Comment:
Since this function is so simple now, can you refactor all of its call sites to call `getRandomPort` directly? (Or do you have a specific reason for keeping it around?)